helm/oaf/values.yaml

```diff
@@ -49,9 +75,37 @@ ph-ee-engine:
         - secretName: oafqa-tls
           hosts:
             - "*.qa.oneacrefund.org"
-    kibanaConfig:
-      kibana.yml: |
-        monitoring.enabled: false
+    extraEnvs:
+      - name: 'ELASTICSEARCH_USERNAME'
+        valueFrom:
+          secretKeyRef:
+            name: elastic-credentials
+            key: username
+      - name: 'ELASTICSEARCH_PASSWORD'
+        valueFrom:
+          secretKeyRef:
+            name: elastic-credentials
+            key: password
+      - name: 'KIBANA_ENCRYPTION_KEY'
+        valueFrom:
+          secretKeyRef:
+            name: kibana
+            key: encryptionkey
+      kibanaConfig:
+        kibana.yml: |
+          monitoring.enabled: false
+          server.ssl:
+            enabled: true
```

Comment:
These flags can be passed as false in base helm chart along with all default values so that we can only flip this switch in example helm charts to activate entire security without having to override other defaults.